src/dispatch/service/service.py

```diff
@@ -143,6 +143,7 @@ def get_all_by_project_id_and_status(
         db_session.query(Service)
         .filter(Service.project_id == project_id)
         .filter(Service.is_active.is_(is_active))
+        .order_by(Service.name)
```

Comment:
Is it ordering them by name ASC or DESC? We may want to change it to `.order_by(asc(Service.name))` and add import `desc` by adding `from sqlalchemy import asc` if DESC (see https://docs.sqlalchemy.org/en/20/core/sqlelement.html#sqlalchemy.sql.expression.asc).

Reply:
If it's already in ASC order then we don't need to do anything.

Reply:
It is in ASC order.